Tasks/VsTest/vstest.ts

```diff
@@ -699,6 +699,11 @@ function invokeVSTest(testResultsDirectory: string): Q.Promise<number> {
                 .then(function (vsVersion) {
                     try {
                         vstestLocation = getVSTestLocation(vsVersion);
+                        vsTestVersionForTIA = getVsTestVersion();
```

Comment:
also ensure if vs version is not supported we r priting the message and following normal path (without runsettings etc